src/main/java/racingcar/controller/CarController.java

```diff
@@ -1,46 +1,63 @@
 package racingcar.controller;
 
-import racingcar.domain.Car;
-import racingcar.service.RacingGame;
+import racingcar.model.Car;
+import racingcar.model.RacingGame;
+import racingcar.utils.Validator;
 import racingcar.view.InputView;
 import racingcar.view.OutputView;
 
-import java.util.*;
+import java.util.List;
 
 import static java.util.stream.Collectors.toList;
 
 public class CarController {
+
+    private final InputView inputView;
+
+    public CarController(InputView inputView) {
+        this.inputView = inputView;
+    }
+
     public void run() {
         List<Car> cars = makeCars(getCarNamesFromUser());
         int count = getCountFromUser();
 
         RacingGame racingGame = new RacingGame(cars, count);
-        racingGame.play();
+
+        OutputView.printCurrentStatus(racingGame.getCars());
+        while (!racingGame.isGameFinished()) {
```

Comment:
```suggestion
        while (!racingGame.isFinished()) {
```

이름은 이정도여도 충분할 것 같아요! 다만, 항상 부정하여 사용된다면 `isNotFinished()`, 또는 이것과 의미가 통하는 다른 긍정 표현으로 만드는 것이 더 가독성을 높이는 방법일 수도 있겠네요.